sklearn/metrics/_ranking.py

```diff
@@ -1112,8 +1112,12 @@ def roc_curve(
 
     thresholds : ndarray of shape (n_thresholds,)
         Decreasing thresholds on the decision function used to compute
-        fpr and tpr. `thresholds[0]` represents no instances being predicted
-        and is arbitrarily set to `np.inf`.
+        fpr and tpr.
```

Comment:
I think I prefer it being at least mentioned in the main description, but I am also okay with it the way it is now.